eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -22,16 +22,45 @@ steps:
   - script: ${{ parameters.buildScript }}
             -restore -build
             -configuration ${{ parameters.buildConfig }}
+            /p:ArchiveTests=true
             /bl:${{ parameters.repoLogPath }}/build.binlog
             $(_OfficialBuildIdArgs)
     displayName: Build
 
   - ${{ if ne(parameters.skipTests, 'true') }}:
     - script: ${{ parameters.dotnetScript }} dotnet-coverage collect
               --settings $(Build.SourcesDirectory)/eng/CodeCoverage.config
+              --output ${{ parameters.repoTestResultsPath }}/NonHelixTests.cobertura.xml
+              "${{ parameters.buildScript }} -testnobuild -test -configuration ${{ parameters.buildConfig }} /bl:${{ parameters.repoLogPath }}/tests.binlog $(_OfficialBuildIdArgs)"
+      displayName: Run non-helix tests
```

Comment:
Is it possible to run the non-helix tests and the helix tests in parallel? There shouldn't be a reason to serialize them, and that could speed up our CI legs.

Reply:
We could add a wrapper project that builds both in parallel from msbuild. Should that be in a follow up PR?

Reply:
Why would we need to build them in parallel? I'm talking about running them in parallel.

Why would we need a wrapper project? Does azdo allow for running 2 pipeline tasks at the same time?

Reply:
By "build" in parallel I meant run the Test, and send-to-helix targets in parallel.

Reply:
> Does azdo allow for running 2 pipeline tasks at the same time?

AFAIK, this isn't supported.

Reply:
Yeah, the thing that is allowed is to run two jobs or stages at the same time, but not steps within a job AFAIK. While I do think that could speed up things a bit, I also think it might be overkill to gain 9 mins or so that it takes in average to run the non helix tests first. I propose keeping things as is, and then perhaps look in the future here to see if this is something we need to invest on.